CMakeLists.txt

```diff
@@ -157,6 +157,9 @@ if(CMAKE_COMPILER_IS_MSVC)
     endif()
     # /MANIFEST:NO") # please, don't disable manifest generation, otherwise crash at start for vs2008
 
+    # Enable warnings, which are disabled by default (see https://learn.microsoft.com/de-de/cpp/preprocessor/compiler-warnings-that-are-off-by-default)
+    string(APPEND CMAKE_CXX_FLAGS_RELEASE " /w34265")
```

Comment:
Why only release flags? How about just adding this to line 146?

Reply:
The release thing is a copy & paste error. In our project set everything via `add_compile_options`, but I didn`t know if it should be modifiable from outside or not. The line is separate from the disabled warnings so you have a grouping, making it more readable. I also considered taking out all the `/wd` options from the line above and also documenting them so it is more obvious which options are explicitly enabled and which are disabled.